src/_gettsim_tests/_policy_test_utils.py

```diff
@@ -70,12 +99,18 @@ def load_policy_test_data(policy_name: str) -> list[PolicyTest]:
 
 
 def get_test_data_as_tree(test_data: NestedDataDict) -> NestedDataDict:
+    info = test_data.get("info", {})
```

Comment:
```suggestion
    test_info = test_data.get("info", {})
```

but put into calling function